packages/twenty-front/src/modules/activities/emails/state/viewableThreadState.ts

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+import { MockedThread } from '@/activities/emails/mocks/mockedThreads';
+
+export const viewableThreadState = atom<MockedThread | null>({
```

Comment:
I'm very surprise about the typing getting out of the mocks!

Reply:
Also, I think this state is not useful, we should rely on apollo and not on the cache.

Reply:
About the typing, this is something you want to discuss with Coco and Raphael to redefine the API?